### PR TITLE
Add bayesian dimensionality to likestats

### PR DESCRIPTION
**Pull request by eatdust**
An extra little calculation to get yet another measure of an effective number of parameters. This is d_G from arXiv:1903.06682. Added in the likestats. I've tested it  returns the right value against the output of "anesthetic" on nested sampling chains.

### CODE REVIEW

**Comment by eatdust** [comment]
Yes, 2*var(loglike) would be nice, and, yes, some output in likeSummary would be useful.

As far as I remember, Complexity was commented out in likesummary because this guy is not fully Bayesian, and, may possibly suffer from errors from the max(like), especially from pure MCMC sampling (https://github.com/cmbant/getdist/pull/2).
I would suggest to let complexity as it is, calculated internally but not logged (because I am still using it...), and, add the Bayesian Model Dimensionality in the likesummary output?

**Comment by cmbant** [comment]
Makes sense, sounds good to me!

File: getdist/types.py [hunk 1]
```diff
@@ -855,7 +855,7 @@ def loadFromFile(self, filename):
         self.meanLogLike = results.get('mean(-Ln(like))', None)
         self.logMeanLike = results.get('-Ln(mean like)', None)
         self.complexity = results.get('complexity', None)
-        self.varLogLike = results.get('2*Var(Ln(like))', None)
+        self.varLogLike = 0.5 * results.get('2*Var(Ln(like))', None)
```
Comment (cmbant) [inline getdist/types.py:858]: does 0.5*None work?

Comment (eatdust) [reply getdist/types.py:858]: in python4, it will ;)

**Pull request merged**